ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -96,9 +168,25 @@ export default class StatusDropdown extends React.Component {
       }
     }
 
+    if (
+      frameworkName === 'browsertime' &&
+      this.anyAlertWithVideoResults(alertSummary)
+    ) {
+      alertSummary = await this.addTaskDetails(
```

Comment:
These 2 awaits effectively double the wait time for the modal. We should run them in parallel, using promises.